www/bejt02/sp2/js/main.js

```diff
@@ -0,0 +1,250 @@
+                        <button class="btn btn-secondary">R</button>
+                        <button class="btn btn-secondary">S</button>
+                        <button class="btn btn-secondary">T</button>
+                        <button class="btn btn-secondary">U</button>
+                        <button class="btn btn-secondary">V</button>
+                        <button class="btn btn-secondary">W</button>
+                        <button class="btn btn-secondary">X</button>
+                        <button class="btn btn-secondary">Y</button>
+                        <button class="btn btn-secondary">Z</button>
+                    </div>
+                    <div class="col-1" style="text-align: right">
+                        <a class="btn btn-primary" id="changePage">💛</a>
+                    </div>
+                </div>
+            </div>
+            <div id="images">
+                
+            </div>
+            `
+        )
+        $("#changePage").click(function () {
+            getFavorites()
+        });
+        $("button").click(function(){
+            window.localStorage.setItem("startDrink",$(this).text());
+            getDrinks()
+        })
+      
+    }
+
+    getMain()
+
+    function getDrinks(){
+        $("#images").empty();
+        var letter =window.localStorage.getItem("startDrink");
+        
+        var src = `<div class="container"  style="text-align:center">
+                        <div class="spinner-border">
+                            <span class="sr-only">Loading...</span>
+                        </div>
+                    </div>`
+        $("#images").html(src)
+
+        src =`<div class="row row-cols-3 row-cols-xs-1">`
+
+        $.ajax({url: "https://thecocktaildb.com/api/json/v1/1/search.php?f="+letter, success: function(result){
+            if(result.drinks){
+                $.each(result.drinks, function (i, drink) {
+                    src +=
+                    `<div class="col mb-4"> 
+                        <div class="card mb-3" style="max-width: 540px;">
+                            <div class="row no-gutters">
+                                <div class="col-md-4">
+                                    <img class="drink-image" src="${drink.strDrinkThumb}" alt="...">
+                                </div>
+                                <div class="col-md-8">
+                                    <div class="card-body">
+                                        <h5 class="card-title">${drink.strDrink}</h5>
+                                        <p class="card-text">${drink.strInstructions}</p>
+                                    </div>
+                                </div>
+                            </div> 
+                            <div class="row no-gutters">
+                                <ul class="list-group list-group-flush">`
+                                    for (let index = 0; index < 16; index++) {
```

Comment:
proc je tady 16?

Reply:
Bylo to kvůli maximálnímu počtu ingrediencí, na základě Vašeho doporučení z hovoru předěláno.